tutorials/Inference-Bert-Model-for-High-Performance-with-ONNX-Runtime-on-AzureML.ipynb

```diff
@@ -0,0 +1,690 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.image import ContainerImage\n",
+    "image_config = ContainerImage.image_configuration(execution_script = \"score.py\",\n",
+    "                                                  runtime = \"python\",\n",
+    "                                                  conda_file = \"myenv.yml\",\n",
+    "                                                  description = \"ONNX Bert-squad-large-uncased Demo\",\n",
+    "                                                  tags = {\"demo\": \"onnx\"}\n",
+    "                                                 )\n",
+    "\n",
+    "\n",
+    "image = ContainerImage.create(name = \"onnx-bert-squad-large-uncased\",\n",
+    "                              models = [model],\n",
+    "                              image_config = image_config,\n",
+    "                              workspace = ws)\n",
+    "\n",
+    "image.wait_for_creation(show_output = True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In case you need to debug your code, the next line of code accesses the log file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(image.image_build_log_uri)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We're all set! Let's get our model chugging.\n",
+    "\n",
+    "## Step 3.5 - Deploy the container image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.webservice import AciWebservice\n",
+    "\n",
+    "aciconfig = AciWebservice.deploy_configuration(cpu_cores = 1, \n",
+    "                                               memory_gb = 4, \n",
+    "                                               tags = {'demo': 'onnx'}, \n",
+    "                                               description = 'web service for Bert-squad-large-uncased ONNX model')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following cell will likely take a few minutes to run as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.webservice import Webservice\n",
+    "from random import randint\n",
+    "\n",
+    "aci_service_name = 'onnx-bert-squad-large-uncased-'+str(randint(0,100))\n",
+    "print(\"Service\", aci_service_name)\n",
+    "\n",
+    "aci_service = Webservice.deploy_from_image(deployment_config = aciconfig,\n",
+    "                                           image = image,\n",
+    "                                           name = aci_service_name,\n",
+    "                                           workspace = ws)\n",
+    "\n",
```

Comment:
Replace **all of this** with:

from azureml.core.model import InferenceConfig
from azureml.core.environment import Environment


myenv = Environment.from_conda_specification(name="myenv", file_path="myenv.yml")
inference_config = InferenceConfig(entry_script="score.py", environment=myenv)

from azureml.core.webservice import AciWebservice

aciconfig = AciWebservice.deploy_configuration(cpu_cores = 1, 
                                               memory_gb = 4, 
                                               tags = {'demo': 'onnx'}, 
                                               description = 'web service for TinyYOLO ONNX model')

aci_service = Model.deploy(ws, aci_service_name, [model], inference_config, aciconfig)
aci_service.wait_for_deployment(True)
print(aci_service.state)